packages/terra-date-time-picker/src/DateTimeUtils.js

```diff
@@ -256,6 +260,14 @@ class DateTimeUtils {
   static isMomentObject(value) {
     return moment.isMoment(value);
   }
+
+  /**
+   * Creates moment object with local timezone.
+   * @return {object|undefined}- The moment object. Undefined if unable to convert.
```

Comment:
This inline doc is inaccurate. `moment.tz.guess()` returns the local timezone string. For example, `America/Chicago`. It doesn't return a moment object.

Reply:
Changed here b5f07bc5f04b127f3d75bc46a3651a2e1091b165